spring-cloud-skipper-server-core/src/test/java/org/springframework/cloud/skipper/server/controller/ReleaseControllerTests.java

```diff
@@ -306,6 +316,33 @@ public DeploymentState getState() {
 
 	}
 
+	@Test
+	public void getFromAndPostToActuator() throws Exception {
+		install("ticktock", "1.0.0", "myTicker");
+		assertReleaseIsDeployedSuccessfully("myTicker", 1);
+
+		mockMvc
+				.perform(get("/api/release/actuator/myTicker/myTicker.log-v1/myTicker.log-v1-0?endpoint=info"))
+				.andExpect(status().isOk()).andReturn();
+
+		verify(actuatorService, times(1))
+			.getFromActuator("myTicker", "myTicker.log-v1", "myTicker.log-v1-0","info");
+
+
+		reset(actuatorService);
```

Comment:
Any reason to not split these into separate tests?

Reply:
I merged them because `install` takes some time.